DESIGN-NOTES.md

```diff
@@ -38,11 +38,11 @@ Instrumentation Engine allows multiple profilers co-exist in the same process. I
 
-Sticking with the Intellitrace terminology, these instrumentation components called "Instrumentation Methods" which will essentially be free threaded com objects loaded by direct call to the class factory using the com server exports (DllGetClassObject etc…). This has the advantage of requiring a simple vtable dispatch for each cross component call keeping overhead to a minimum.
+Sticking with the IntelliTrace terminology, these instrumentation components called "Instrumentation Methods" which will essentially be free threaded com objects loaded by direct call to the class factory using the com server exports (DllGetClassObject etc…). This has the advantage of requiring a simple vtable dispatch for each cross component call keeping overhead to a minimum.
 
 Since ICorProfiler reserves the right to callback on multiple threads simultaneously, the instrumentation methods will need to act as free-threaded objects and take responsibility for their own locking. This is nothing new, as ICorProfiler has always had that requirement.
 
-In order to allow for xcopy deploy without admin rights, the registration of the instrumentation components is done via an object configuration model using xml files. This is similar to how intellitrace manages instrumentation methods today with one key difference: the instrumentation objects will likely be packaged in their own dlls. Therefore, for MSI based installations, a safe admin writable folder will contain the dlls, and the configuration file will describe the objects that are contained within that dll. For xcopy deployment, the sub folder containing the instrumentation methods will be a sub-folder of the xcopy payload and the xml config will be contained in the same folder as the monitoring agent.
+In order to allow for xcopy deploy without admin rights, the registration of the instrumentation components is done via an object configuration model using xml files. This is similar to how IntelliTrace manages instrumentation methods today with one key difference: the instrumentation objects will likely be packaged in their own dlls. Therefore, for MSI based installations, a safe admin writable folder will contain the dlls, and the configuration file will describe the objects that are contained within that dll. For xcopy deployment, the sub folder containing the instrumentation methods will be a sub-folder of the xcopy payload and the xml config will be contained in the same folder as the monitoring agent.
```

Comment:
>ntelliTrace manages instrumentation meth [](start = 191, length = 40)

This is an internal implementation detail of IntelliTrace, and it doesn't really map to what happens in CLRInstrumentationEngine. I would take out references to IntelliTrace.